charts/vald/templates/index/job/correction/networkpolicy.yaml

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
> **NOTE**
 It might be better to add `NetworkPolicy` resource since job components(`correction/creation/save`) are also a Vald component. 🤔